common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -78,6 +80,27 @@
   public static final String MERGE_DIR_KEY = "mergeDir";
   public static final String ATTEMPT_ID_KEY = "attemptId";
   private static final int UNDEFINED_ATTEMPT_ID = -1;
+  private static final int UNDEFINED_SHUFFLE_MERGE_ID = Integer.MIN_VALUE;
+
+  // ConcurrentHashMap doesn't allow null for keys or values which is why this is required.
```

Comment:
The comment is outdated.

Reply:
Can you please explain why is this outdated?

Reply:
Do we still use the `ConcurrentHashMap`?

Reply:
Yes, we are still using `ConcurrentHashMap` for the `shuffleMergePartitionsByMergeId` right which cannot take null keys or values which is why this marker unmodifiable Map is required. This is not really about the `ConcurrentHashMap` marker rather the `ConcurrentHashMap` used for `shuffleMergePartitionsByMergeId`